go-controller/pkg/util/gateway-init.go

```diff
@@ -136,18 +135,16 @@ func GatewayInit(clusterIPSubnet []string, nodeName, nicIP, physicalInterface,
 		gatewayRouter, "--", "set", "logical_router", gatewayRouter,
 		"options:chassis="+systemID, "external_ids:physical_ip="+physicalIP)
 	if err != nil {
-		logrus.Errorf("Failed to create logical router %v, stdout: %q, "+
+		return fmt.Errorf("Failed to create logical router %v, stdout: %q, "+
```

Comment:
Can we use `pkg.errors/Errwrap` here? it's the new idiom.

Reply:
@squeed yeah, could do, but I'd rather as a separate follow-up commit to reduce the actual diff